ui/StatusQ/src/typesregistration.cpp

```diff
@@ -77,6 +78,7 @@ void registerStatusQTypes() {
     qmlRegisterSingletonType<UrlUtils>("StatusQ", 0, 1, "UrlUtils", [](QQmlEngine* engine, QJSEngine*) {
         return new UrlUtils(engine);
     });
+    qmlRegisterSingletonType<AudioUtils>("StatusQ", 1, 0, "AudioUtils", &AudioUtils::qmlInstance);
```

Comment:
Detail: for consistency we could create in-place factory function as for e.g. UrlUtils above instead of using `qmlInstance` from the class.

Reply:
True but the whole C++ class is not needed at all imo